frame/balances/e2e-tests/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "pallet-balances-e2e-tests"
+version = "2.0.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
+publish = false
+
+[dependencies]
+pallet-sudo = { version = "2.0.0", path = "../../../frame/sudo" }
+sp-keyring = { version = "2.0.0", path = "../../../primitives/keyring" }
+substrate-test-runner = { git = "https://github.com/paritytech/substrate-test-runner", branch = "substrate" }
```

Comment:
is `substrate` supposed to be a long-term-supported branch? shouldn't we have this point to `master`/`main`?

Reply:
yeah I have a PR open there, once we finalize things here, will have that merged to master